tests/ui/mixed_attributes_style.rs

```diff
@@ -0,0 +1,39 @@
+#![warn(clippy::mixed_attributes_style)]
+
+#[allow(unused)] //~ ERROR: item has both inner and outer attributes
+fn foo1() {
+    #![allow(unused)]
+}
+
+#[allow(unused)]
+#[allow(unused)]
```

Comment:
Attribute duplication might be worth another lint.

Reply:
Dully noted.